### PR TITLE
Fix naming issues with combine files

### PR DESCRIPTION
**Pull request by jwalsh28**
-	Rate arrests race groups (should not include white, non-Hispanic) 
-  Communities – maybe try “Neighborhoods”

### CODE REVIEW

File: 10_construct-database/61_construct_place_all.qmd
Comment (cdsolari) [inline 10_construct-database/61_construct_place_all.qmd:1]: Can you change the "asian_other" to share_desc_rep_other, share_desc_rep_other_quality in all your files? This is what we told Graphicacy to expect. If you feel strongly about it, we can communicate this change and it shouldn't be too late!

**Review by cdsolari — CHANGES_REQUESTED** [review]
We still have places in several files where the "asian_other" isn't changed to share_desc_rep_other, share_desc_rep_other_quality. Is that something we can change? I also know that Aaron made a last minute decision to change all the racial sub-area labels to something like white communities, communities of color, and mixed communities. Can we chat about this? It is lowest priority though!

**Review by cdsolari — APPROVED** [review]
Looks like all the intended changes worked!

**Comment by jwalsh28** [comment]
Great! Leaving this PR open for now just in case we stumble across anyhting else as we are finalizing tests.

**Pull request merged**